WebPhapp/backend/server.js

```diff
@@ -460,7 +460,7 @@ Returns:
 Relevant Express Docs:
     https://expressjs.com/en/api.html#req.query
 */
-app.get('/api/v1/patients', (req,res) => {
+app.get('/api/v1/patients', auth.checkAuth([Role.Prescriber, Role.Government, Role.Dispenser]), (req,res) => {
```

Comment:
Patients should be able to see information about themselves? Or, are we not allowing this ?